includes/template-functions.php

```diff
@@ -770,11 +770,13 @@ function sensei_quiz_has_questions() {
 
 	global $sensei_question_loop;
 
-	if ( ! isset( $sensei_question_loop['total'] ) ) {
+	$questions_count = count( $sensei_question_loop['questions'] );
```

Comment:
The total count is no longer relevant because it contains the count of all the questions, but for the loop, we need the count of the questions for the current page.